rcsb/utils/dictionary/DictMethodResourceCacheWorkflow.py

```diff
@@ -38,16 +41,19 @@ def __init__(self, **kwargs):
         Args:
             configPath (str, optional): path to configuration file (default: exdb-config-example.yml)
             configName (str, optional): configuration section name (default: site_info_configuration)
-            cachePath (str, optional):  path to cache directory (default: '.')
+            mockTopPath (str, optional):  mockTopPath is prepended to path configuration options if it specified (default=None)
+            workPath (str, optional):  path to working directory (default: HERE)
+            cachePath (str, optional):  path to cache directory (default: HERE/CACHE)
             stashRemotePrefix (str, optional): file name prefix (channel) applied to remote stash file artifacts (default: None)
+            debugFlag (bool, optional):  sets logger to debug mode (default: False)
         """
         configPath = kwargs.get("configPath", "exdb-config-example.yml")
         self.__configName = kwargs.get("configName", "site_info_configuration")
```

Comment:
Huh, do you know why `DictMethodResourceCacheWorkflow` uses `site_info_configuration`, whereas `NeighborInteractionWorkflow` originally had `site_info_remote_configuration`? 

Since we use the latter in production, I wonder if we can start transitioning to that? https://github.com/rcsb/rcsb-configs/blob/master/profiles/parks/prod/east/A/exdb-loader-config.yml

Also, that's the configuration file that we'll be using in the Luigi workflow, so we'll want to make sure we specify the correct configName.

Reply:
Yes, during this whole effort I've noticed that the default `configName` inconsistent in these Workflow classes but it is often `site_info_configuration` including `ExDbWorkflow` and `RepoLoadWorkflow`, I assume those are getting passed in by Luigi workflow. In my testing, I've been explicitly passing in `site_info_remote_configuration` so it hasn't been an issue.

I'll change to `site_info_remote_configuration` for these two files, though since that change for neighbors was not on purpose. I'm going to make the change for ProteinTargetSequenceExecutionWorkflow also since I noticed when making this change that I did not allow cachePath to be overridden for that one, just workPath.

Reply:
Sounds good, thanks. Yeah, I think we'll need to start switching all other repos to use `site_info_remote_configuration` as we move forward. I think the `site_info_configuration` is just leftover from before we switched over to HTTP-based file access.

Reply:
Sorry you're getting stuck with some of this historical baggage...appreciate you helping to bring them to light and resolve them :)